CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Prowide Core - CHANGELOG
 
+#### 9.4.19 - SNAPSHOT
```

Markdownlint

Expected: h2; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)